### PR TITLE
proposal for fabric-multi-channel-network-samples

### PR DESCRIPTION
**Pull request by nicezic0**
# Lab Name
Fabric Multi Channel Network Samples

# Short Description
This lab provides samples for multi channel network using Hyperledger Fabric.
## Directory Info
* bin : The built binaries, after scripts/bootstrap.sh, which needed to run for the first time.
* chaincode : Chaincodes for the samples.
* multi-channel-network : The first sample based on 'BYFN' in 'fabric-samples' repo.
* fabric-ca : The second sample based on 'fabric-ca' in 'fabric-samples' repo.

# Scope of Lab
This lab is based on [Build Your First Network](https://github.com/hyperledger/fabric-samples).

After the [tutorial](https://hyperledger-fabric.readthedocs.io/en/latest/build_network.html), a Fabric beginner may want to find out multi channel network samples, and these are the samples.
Currently, There are two multi-channel-network samples from 'byfn' and 'fabric-ca' both are based on 'fabric-samples' repo.

We will update more samples.

# Initial Committers
- https://github.com/nicezic0 (maintainer)

# Sponsor
- https://github.com/christo4ferris (Christopher Ferris)

# Pre-existing repository
- https://github.com/nicezic0/fabric-multi-channel-samples

Signed-off-by: YeonhoPark <nicezic@naver.com>

### CODE REVIEW

**Comment by christo4ferris** [comment]
LGTM thanks!

**Comment by tkuhrt** [comment]
@hyperledger-labs/lab-stewards : Any thoughts on this one? 

@nicezic0 : Can we get more descriptive in the scope of the lab? Will we just be taking the existing BYFN samples and adding a second channel to it? Why not include this in the existing Fabric Samples?

**Comment by nicezic0** [comment]
@tkuhrt Hello.
I did pull-request this on the existing 'fabric-samples' by Gerrit, and @christo4ferris mentioned bellow.

> YeonHo, thanks for this contribution. Appreciate the effort. However, I think that this would be better handled as a Hyperledger Lab project. We are trying to keep the samples limited to the tutorials in the documentation as keeping them all up-to-date is quite an undertaking.


Here's the link. https://gerrit.hyperledger.org/r/#/c/26946
Thanks.

**Comment by lehors** [comment]
I agree with @tkuhrt that the description could be improved a bit. It wouldn't hurt to expand on those 2 sentences. :-)
I have otherwise no concerns. It would fit nicely within Fabric and I hope we can get it there eventually but for now this seems to be a good first step.
+1

**Comment by nicezic0** [comment]
@tkuhrt @lehors I updated the description. 
Thank you very much.

**Comment by ryjones** [comment]
+1

**Comment by nicezic0** [comment]
FYI, There was a conflict and now solved.

File: labs/fabric-multi-channel-network-samples.md [hunk 1]
```diff
@@ -0,0 +1,28 @@
...
+* bin : The built binaries, after scripts/bootstrap.sh, which needed to run for the first time.
+* chaincode : Chaincodes for the samples.
+* multi-channel-network : The first sample based on 'BYFN' in 'fabric-samples' repo.
+* fabric-ca : The second sample based on 'fabric-ca' in 'fabric-samples' repo.
+
+# Scope of Lab
+This lab is based on [Build Your First Network](https://github.com/hyperledger/fabric-samples).
+
+After the [tutorial](https://hyperledger-fabric.readthedocs.io/en/latest/build_network.html), a Fabric beginner may want to find out multi channel network samples, and these are the samples.
+Currently, There are two multi-channel-network sampels from 'byfn' and 'fabric-ca' both are based on 'fabric-samples' repo.
```
Comment (tkuhrt) [inline labs/fabric-multi-channel-network-samples.md]: sampels -> samples

Comment (nicezic0) [reply labs/fabric-multi-channel-network-samples.md]: @tkuhrt My mistake. Sorry.

**Comment by tkuhrt** [comment]
Looks like we have two +1's on this PR. Going forward with creating this lab. I will close this when the lab is created.

**Comment by tkuhrt** [comment]
Created Lab https://github.com/hyperledger-labs/fabric-multi-channel-network-samples

**Pull request merged**